test/terraform-aws-route53-query-logs_test.go

```diff
@@ -0,0 +1,50 @@
+package test
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/aws"
+	"github.com/gruntwork-io/terratest/modules/random"
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	test_structure "github.com/gruntwork-io/terratest/modules/test-structure"
+)
+
+func TestTerraformRoute53QueryLog(t *testing.T) {
+	t.Parallel()
+
+	tempTestFolder := test_structure.CopyTerraformFolderToTemp(t, "../", "examples/simple")
+
+	// Give this S3 Bucket a unique ID for a name tag so we can distinguish it from any other Buckets provisioned
+	// in your AWS account
+	testName := fmt.Sprintf("terratest-aws-route53-query-logs-%s", strings.ToLower(random.UniqueId()))
+	logGroupName := fmt.Sprintf("/aws/route53/%s.com.", testName)
+	logStreamName := "route53-test-log-stream"
+	awsRegion := "us-east-1"
```

Comment:
Can you add a comment that this must be us-east-1 because there's the only place route53 query logs are available?